src/main/java/at/hannibal2/skyhanni/config/features/inventory/MagicalPowerConfig.java

```diff
@@ -0,0 +1,19 @@
+package at.hannibal2.skyhanni.config.features.inventory;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class MagicalPowerConfig {
+    @Expose
+    @ConfigOption(name = "Magical Power Display", desc = "Show Magical Power as stack size inside Accessory Bag and Auction House.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enabled = false;
+
+    @Expose
+    @ConfigOption(name = "Colored", desc = "Whether to make the numbers colored.")
+    @ConfigEditorBoolean
+    public boolean colored = false;
```

Comment:
We use the spelling "colour" not "color".
You also need to change it in at the places where it is used,
```suggestion
    @Expose
    @ConfigOption(name = "Coloured", desc = "Whether to make the numbers coloured.")
    @ConfigEditorBoolean
    public boolean coloured = false;
```

Reply:
I take that back. I guess hannibal changed his mind about that. Leave it as is.

Reply:
With the planned gb/us English setting, the words in the config and the search doesn't matter for the user-facing part. In the backend, I still prefer color over colour, for consistency:
![image](https://github.com/user-attachments/assets/547896c5-8919-4e9f-b210-7c2570461b17)
![image](https://github.com/user-attachments/assets/f01769f5-a871-40d4-a10d-2b66b992dccb)

Reply:
Related info: https://github.com/hannibal002/SkyHanni/pull/2258